Library/ViewModels/CancelPledgeViewModelTests.swift

```diff
@@ -222,4 +222,19 @@ final class CancelPledgeViewModelTests: TestCase {
       self.cancelPledgeError.assertValues(["You can't cancel your pledge right now."])
     }
   }
+
+  func testTrackingEvents() {
+    let mockService = MockService(cancelBackingResult: .init(success: .init()))
+
+    withEnvironment(apiService: mockService) {
+      self.vm.inputs.configure(with: .template, backing: .template)
+      self.vm.inputs.viewDidLoad()
+
+      self.cancelPledgeButtonEnabled.assertValues([true])
```

Comment:
The cancelPledgeButton status is already being tested in other place and it's not so useful here since you're only testing the tracking. Instead, I would replace this line with `XCTAssertEqual([], self.trackingClient.events)` to prove that the event is only tracked after the button is tapped.